### DISCUSSION TITLE
installation without tests

### DISCUSSION THREAD

**Discussion by vermaete** [python-jsonschema/jsonschema#975] [General]
Hi,

Is there a way to install this without the tests directory?
Just because I don't need them at my tiny IOT device?

br

**Comment by Julian** [comment]
Not natively but you could always just remove them.

**Reply by vermaete** [reply]
Could it become a todo?  I don't know if there a common way to do in Python.

**Reply by Julian** [reply]
It's kind of nontrivial packaging complexity for a rare case -- the files are pretty tiny:

```
⊙  du -sh jsonschema/tests                                                                                                                                                          julian@Airm
180K	jsonschema/tests
```

so I don't think this is something I'd want to add honestly.

**Comment by vermaete** [comment]
Hi,

I  solved it in the Yocto/OE recipe:
https://cgit.openembedded.org/openembedded-core/tree/meta/recipes-devtools/python/python3-jsonschema_4.7.2.bb?h=master#n12

Thanks

On Thu, Jul 28, 2022 at 1:41 PM Julian Berman ***@***.***>
wrote:

> It's kind of nontrivial packaging complexity for a rare case -- the files
> are pretty tiny:
>
> ⊙  du -sh jsonschema/tests                                                                                                                                                          ***@***.***
>
> 180K	jsonschema/tests
>
>
> so I don't think this is something I'd want to add honestly.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/python-jsonschema/discussions/975#discussioncomment-3268971>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5IFKVQCECUI2GMRAR77TDVWJWVXANCNFSM53YPAJ6Q>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***
> .com>
>


-- 
Jan Vermaete
“Success is a self-correcting phenomenom.” -- Gary Hamel

**Reply by Julian** [reply]
Nice, glad to hear it.
